### PR TITLE
#23 Feat/common

### PR DESCRIPTION
**Pull request by LanHuong142002**
Common CSS:

- abstracts
  - all.css
  - variables.css
- base
  - all.css
  - reset.css
  - typography.css

This PR will block task #17

### CODE REVIEW

**Review by GraphicDThanh — APPROVED** [review]

File: practice-one/styles/abstracts/variables.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+:root {
+  /* Color */
+  --normal-color: #ff5300;
+  --normal-color-5: #e64d00;
```
Comment (phuongbuimai-agilityio) [inline practice-one/styles/abstracts/variables.css:4]: Like the class name, the variable shouldn't have a number on it

File: practice-one/styles/abstracts/variables.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  --light-color: #fff7f4;
+  --white: #ffffff;
+  --secondary-color: #7b7b7b;
```
Comment (phuongbuimai-agilityio) [inline practice-one/styles/abstracts/variables.css:7]: Where is the primary-color?

File: practice-one/styles/abstracts/variables.css [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  --dark-color: #373737;
+
+  /* Font-large */
+  --huge: 65px;
+  --biggest: 49px;
+  --large: 39px;
+  --almost-small: 25px;
+  --small: 20px;
+  --small-medium: 13px;
```
Comment (phuongbuimai-agilityio) [inline practice-one/styles/abstracts/variables.css:16]: Should choose another name for it

File: practice-one/styles/abstracts/variables.css [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  --smallest: 10.2px;
```
Comment (phuongbuimai-agilityio) [inline practice-one/styles/abstracts/variables.css:17]: Should choose the integer number

**Pull request merged**